proto/v1/auth_service.proto

```diff
@@ -163,10 +164,12 @@ message User {
 
   string password = 5 [(google.api.field_behavior) = INPUT_ONLY];
 
-  UserType user_type = 6;
+  google.protobuf.StringValue idp_name = 6;
```

Comment:
// The identity provider name. Example: idps/gitlab.
string identity_provider = 6 [(google.api.field_behavior) = OUTPUT_ONLY];

Reply:
As it is nullable, maybe we should use `google.protobuf.StringValue` instead of `string`?

Reply:
Empty string is null.

Reply:
Btw, this should be called "idp" instead of idp_name. Similar to project in Database.

https://google.aip.dev/124

Reply:
Done